client/src/modules/inventory/configuration/groups/groups.html

```diff
@@ -22,18 +22,31 @@
           <th translate> FORM.LABELS.NAME </th>
           <th translate> INVENTORY.CONTAINS </th>
           <th class="text-center" colspan="3" translate> FORM.LABELS.ACCOUNT </th>
-          <th>&nbsp;</th>
+          <th colspan="2">&nbsp;</th>
         </tr>
         <tr>
-          <th></th><th></th><th></th>
+          <th></th>
+          <th>
+            <div class="input-group">
+              <span class="input-group-addon"><i class="fa fa-filter"></i></span>
+              <input
+                type="text" 
+                ng-model="GroupsCtrl.searchValue"
+                type="search"
+                class="form-control input-sm"
+                style="display: inline"
+                placeholder="{{ 'FORM.LABELS.SEARCH' | translate }}">
```

Comment:
Just so you know, when you are using regular HTML inputs (`<input>`), you can get a performance increase by using [`translate-attr`](https://angular-translate.github.io/docs/#/api/pascalprecht.translate.directive:translate-attr).  This is similar to using the directive instead of the filter, as it reduces `$watch` statements.

In this case, it would be:
```html
translate-attr="{ placeholder: 'FORM.LABELS.SEARCH' }"
```